facebook_connector.py

```diff
@@ -0,0 +1,110 @@
+"""
+
+Author: Nigel Schuster <nigel schusters at gmail dot com>
+
+This module provides a simple abstraction for accessing the Facebook chat data
+
+"""
+
+from __future__ import print_function
+
+from bs4 import BeautifulSoup
+from fbchat_archive_parser.parser import MessageHtmlParser
+import os
+import pandas as pd
+import re
+import warnings
+
+_messages_file = None
+_fb_id_pattern = re.compile(r"(\d+)@facebook\.com")
```

Comment:
Remove, this isn't used (yet).  Also when you add it back in the later PR make it _FB_ID_PATTERN since it's a constant.